File: PRAK205/PRAK205-2310817110011-GHANI_MUDZAKIR.c
#include <stdio.h>
#include <math.h>

int main() {
    //operasi input 
    int a, b, c; 
    scanf("%d %d", &a, &b);
    double a_pangkat = pow(a, 2);
    double b_pangkat = pow(b, 2);

    //operasi mencari c
    double alas = b_pangkat-a_pangkat;
    c = sqrt(alas);      

    int keliling;
    int luas;

    keliling = a + b + c;
    luas = 1/(float)2 * c * a;

    printf("\n\nAlas : %d cm \n", c); 
    printf("Tinggi : %d cm \n", a );
    printf("Keliling : %d  cm \n", keliling);
    printf("Luas : %d  cm^2 \n", luas);


    return 0;
}

File: PRAK205/PRAK205-2310817110011-GHANI_MUDZAKIR.py
import math

a,b = map(int, input().split())

a_pangkat = pow(a, 2)
b_pangkat = pow(b, 2)
alas = b_pangkat-a_pangkat
c = math.sqrt(alas)

keliling = a+b+c
luas = float(1/2) * c * a

print(f"Alas : {int(c)} cm")
print(f"Tinggi : {a} cm")
print(f"Keliling : {int(keliling)} cm")
print(f"Luas : {int(luas)}  cm^2")


